crates/holochain_types/src/app.rs

```diff
@@ -257,8 +257,7 @@ pub enum RoleSettingsYaml {
 #[serde(tag = "type", content = "value", rename_all = "snake_case")]
 pub enum AppBundleSource {
     /// The raw bytes of an app bundle
-    #[serde(with = "serde_bytes")]
-    Bytes(Vec<u8>),
+    Bytes(bytes::Bytes),
```

Comment:
The primary change, everything else is downstream of this.

If there is a preference to write this as `Bytes` and put the import at the top of the file, LMK.

Reply:
Just noticing the removal of serde_bytes here... Does the Bytes type automagically encode as bytes in serde?

Reply:
Yes, with the additional feature flag "serde"

Reply:
realized `serde_bytes` is no longer used anywhere in these crates, removed the dep: https://github.com/holochain/holochain/pull/4891/commits/90f1574c648fabdf06a952a76db354fc0b15740f